pkg/api/v1/conversion_generated.go

```diff
@@ -41,7 +41,7 @@ func convert_api_Binding_To_v1_Binding(in *api.Binding, out *Binding, s conversi
 	if defaulting, found := s.DefaultingInterface(reflect.TypeOf(*in)); found {
 		defaulting.(func(*api.Binding))(in)
 	}
-	if err := convert_api_TypeMeta_To_v1_TypeMeta(&in.TypeMeta, &out.TypeMeta, s); err != nil {
+	if err := s.Convert(&in.TypeMeta, &out.TypeMeta, 0); err != nil {
```

Comment:
Hm. I think you exposed a problem in the conversion generator. It should keep these functions. It will still work as it is but there may be a performance regression. It might be OK to fix this in a followup PR. @wojtek-t 

(Thanks to @caesarxuchao for pointing this out)